test/e2e/app/state.go

```diff
@@ -169,7 +160,7 @@ func (s *State) Commit() (uint64, []byte, error) {
 }
 
 func (s *State) Rollback() error {
-	bz, err := os.ReadFile(s.previousFile)
+	bz, err := ioutil.ReadFile(s.previousFile)
```

Comment:
Should we consider bumping the Go toolchain version in the v0.35 series?

Reply:
Sure. I don't really have a strong opinion on it.

Reply:
I only asked because Go ≥ 1.16 supports `os.ReadFile`, so we only need to use `ioutil` if we expect to support older versions of the toolchain.

Reply:
I wasn't sure if it was 1.16 or 1.17 hence I changes these to use ioutil (also because that's what I had to do for 0.34). I can revert this though. I also realized we're not even using 1.17 yet

Reply:
Yeah, it's not always obvious. The trick I use is to look on godoc.org, where they now have "added in vXX" notations on package entries, e.g., https://pkg.go.dev/os#ReadFile